pyproject.toml

```diff
@@ -51,7 +52,7 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.9"
-docopt-ng = ">=0.9.0"
+typer = ">=0.15.2"
```

Comment:
Our existing matrix of distributions means we need to support as far back as `0.7.0` because that's what Debian 12 shipped with.

Reply:
ok, thanks and done